PLOS/rhino
# AMBR-825

*JIRA issue:* https://jira.plos.org/jira/browse/AMBR-825

*Related PRs:* 

https://github.com/PLOS-Formulas/rhino-formula/pull/92

## What this PR does:

Upgrades tomcat to 8.5

## Developer Notes

To test:

1. Checkout the `AMBR-825` branch in `rhino-formula`.
2. Checkout the `AMBR-825` branch in `rhino`
3. Run `mvn clean package` in `rhino`
4. Run `pogos vigor vagrant up apps/ambra`
5. Follow steps in https://github.com/PLOS/molten/blob/dev/vagrant/apps/ambra/README.md to test the ambra stack.
6. Ensure that `mvn cargo:run` correctly starts a local rhino server for you on http://localhost:8082/v2/

# Code Author Tasks:

> This list should be finished before code review:
- [x] Ticket AC is updated with any decisions made in comments or side conversations.
- [x] Testing or PO Accept notes that will assist the tester, Product Owner or reviewer are set in the ticket.
- [x] All PRs are created and linked for work I have done against other projects as part of this ticket.
- [x] Intergration Tests have been addressed.  QA has been notified of the feature and a ticket has been created for that work, or integration tests have been updated. 
- [x] There are unit tests sufficient for both positive and negative test cases, test coverage has not decreased as part of this work. 
- [x] Any necessary documentation in confluence or READMEs is updated.

# Code Reviewer Tasks
- [ ] I ran the code (in the review environment or locally). I agree the running code fulfills the Acceptance Criteria as stated on the JIRA ticket. If not practical I have seen some evidence that the code does what it says (tests have passed).
